src/theory/quantifiers/fmf/bounded_integers.cpp

```diff
@@ -750,8 +758,8 @@ Node BoundedIntegers::getSetRangeValue( Node q, Node v, RepSetIterator * rsi ) {
   //   e.g.
   // singleton(0) union singleton(1)
   //   becomes
-  // C1 union ( witness y. card(S)<=1 OR ( y in S AND distinct( y, C1 ) ) )
-  // where C1 = ( witness x. card(S)<=0 OR x in S ).
+  // C1 union (witness y. S =(set.singleton C1) OR (y in S AND distinct(y, C1)))
```

Comment:
`C1` is an element, not a set, so we can't apply union to C1. May be `(set.singleton C1)`.